core/services/ardupilot_manager/firmware_download/FirmwareDownload.py

```diff
@@ -20,6 +23,12 @@
     ssl._create_default_https_context = ssl._create_unverified_context
 
 
+def get_sitl_platform() -> str:
```

Comment:
maybe "get_sitl_binary_name" or something?

Reply:
Yup

Reply:
Maybe "get_sitl_platform_name"?

The platform is a object property, so I like the idea of maintaining it's name on the function.

Reply:
sunds good.